adls-plugins/docs/AzureDataLakeStore-batchsink.md

```diff
@@ -19,8 +19,11 @@ Properties
 **fileSystemProperties:** A JSON string representing a map of properties
 needed for the distributed file system. (Macro-enabled)
 
-**schema:** Output schema for the JSON document.
+**schema:** Output schema for the avro format JSON document
```

Comment:
I would move outputFormat before schema, so people read about formats before reading about format specific properties.

Reply:
Output schema of the JSON document. I think it still makes sense to have schema for text and not just avro. In the text case, somebody may try to drop fields from the schema if they don't want them to be written.

Reply:
should mention that certain types are not allowed when using the text format, and an empty schema means the schema of input records will be used.